client/state/stats/lists/selectors.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import get from 'lodash/get';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import i18n from 'lib/mixins/i18n';
```

Comment:
Do we need to use the `moment` from `i18n`, or could we import `moment` directly?